clients/client-go/README.md

```diff
@@ -1,44 +1,43 @@
-[![GoDoc](https://godoc.org/github.com/taskcluster/taskcluster/clients/client-go?status.svg)](https://godoc.org/github.com/taskcluster/taskcluster/clients/client-go)
+[![GoDoc](https://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/v14?status.svg)](https://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/v14)
 [![Coverage Status](https://coveralls.io/repos/taskcluster/taskcluster/clients/client-go/badge.svg?branch=master&service=github)](https://coveralls.io/github/taskcluster/taskcluster/clients/client-go?branch=master)
 [![License](https://img.shields.io/badge/license-MPL%202.0-orange.svg)](http://mozilla.org/MPL/2.0)
 
 A go (golang) port of taskcluster-client.
 
-Complete godoc documentation [here](https://godoc.org/github.com/taskcluster/taskcluster/clients/client-go).
+Complete godoc documentation [here](https://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/v14).
 
 This library provides the following packages to interface with Taskcluster:
 
 ### HTTP APIs
-* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/tcauth
-* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/tcawsprovisioner
-* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/tcec2manager
-* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/tcgithub
-* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/tchooks
-* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/tcindex
-* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/tclogin
-* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/tcnotify
-* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/tcpurgecache
-* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/tcqueue
-* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/tcsecrets
+* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/v14/tcauth
+* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/v14/tcawsprovisioner
+* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/v14/tcec2manager
+* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/v14/tcgithub
+* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/v14/tchooks
+* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/v14/tcindex
+* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/v14/tclogin
+* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/v14/tcnotify
+* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/v14/tcpurgecache
+* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/v14/tcqueue
+* http://godoc.org/github.com/taskcluster/taskcluster/clients/client-go/v14/tcsecrets
```

Comment:
I guess now that we have the monorepo, it makes sense that the services are versioned together with the platform, rather than having individual version numbers, so this seems ok. I just wanted to highlight that until now, services were versioned independently (so e.g. `tcqueue` could have a different version number to `tcsecrets`) but I guess that doesn't make much sense for us now, so this looks fine.

Reply:
That could still happen -- a single application could certainly link to v14 and v15 in different places.  But tcsecrets v14 and v15 might be identical if the breaking change did not involve that service.